kits/find-your-hotel/apps/.env.example

```
@@ -0,0 +1,5 @@
+LAMATIC_PROJECT_ENDPOINT="Your Lamatic GraphQL endpoint"
+LAMATIC_PROJECT_ID="Project ID from Lamatic Studio"
+LAMATIC_PROJECT_API_KEY="API key from Lamatic Studio"
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LAMATIC_PROJECT_API_KEY key should go before the LAMATIC_PROJECT_ENDPOINT key

(UnorderedKey)